tests/test_freqtradebot.py

```diff
@@ -6302,7 +6286,7 @@ def test_position_adjust2(mocker, default_conf_usdt, fee) -> None:
 
     trade = Trade.session.scalars(select(Trade)).first()
     assert trade
-    assert trade.open_order_id is None
+    assert trade.open_entry_or_exit_orders_count == 0
```

Comment:
You might wanna introduce a new property in `has_open_orders`.
this is quite a common check i think - so it'll be handy (even if we might only use it in tests).

Reply:
Yes but in that case it could be more like `has_entry_or_exit_orders` as some time we don't want to take stop loss in account.

Reply:
well i'd exclude stop orders anyway.
They need a separate property ;)